src/calculating.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculating = (number, answer) => {
```

Comment:
For naming variables and functions, it is better to use names that will tell in more detail what is happening in the code (For example: `calculatingBullsAndCows`, `checkValues` etc. ).